addons/l10n_ro/i18n/ro.po

```diff
@@ -0,0 +1,144 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* l10n_ro
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 15.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-02-16 16:39+0000\n"
+"PO-Revision-Date: 2022-02-16 16:39+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. modules: l10n_ro
+#: model:ir.model,name:l10n_ro.model_res_partner
+msgid "Contact"
+msgstr ""
```

Comment:
Maybe we can translate these? It is not on Transifex yet.
Otherwise this file is useless